tests/ui/specs/TimePicker/Agate-TimePicker-Page.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const {element, Page} = require('@enact/ui-test-utils/utils');
+
+class PickerInterface {
+	constructor (id) {
+		this.id = id;
+	}
+
+	focus () {
+		return browser.execute((el) => el.focus(), $(`#${this.id}>div`));
+	}
+
+	get self () {
+		return element(`#${this.id}`, browser);
+	}
+
+	get hour () {
+		return element('.TimePicker_TimePicker_hourPicker', this.self);
+	}
+	get meridiem () {
+		return element('.TimePicker_TimePicker_meridiemPicker', this.self);
+	}
+	get minute () {
+		return element('.TimePicker_TimePicker_minutePicker', this.self);
+	}
+
+	decrementer (picker) {
+		return element('.internal_Picker_Picker_itemDecrement', picker);
+	}
+	incrementer (picker) {
+		return element('.internal_Picker_Picker_itemIncrement', picker);
+	}
+	active (picker) {
+		return element('.internal_Picker_Picker_active', picker);
+	}
+}
+
+class AgateTimePickerPage extends Page {
+	constructor () {
+		super();
+		this.title = 'TimePicker Test';
+		this.components = {};
```

Comment:
If you refer the Picker instances in `specs.js` files as `this.components.timePickerDefault`, `this.components.timePickerWithDefaultValue` etc., I don't think this is needed.